### ISSUE TITLE
Setup PostgreSQL Database using Prisma ORM

### ISSUE THREAD

**Issue by devvspaces** [devvspaces/collabspace#2]
Create base schema for user onboarding models

**Comment by Preetraj2002** [comment]
Assign me this task. I will do it @devvspaces

**Comment by devvspaces** [comment]
@Preetraj2002 are you still free to work on this

**Comment by Preetraj2002** [comment]
Yup now I am free to work on it ... I had my exams so was not able to do much

**Comment by devvspaces** [comment]
@Preetraj2002 u need to first work on the backend project setup #1

**Comment by devvspaces** [comment]
Should I assign you to that

**Comment by Preetraj2002** [comment]
hey @devvspaces i am new to this postgres and prisma and have very little experience with node.js. Can you pair me up with someone who can help me or give me a couple of days to figure it out and then u can assign me this issue. I hope you understand.

**Comment by devvspaces** [comment]
@Preetraj2002 there is no one I can pair you with now. But have you gotten an idea about this. I can point you to online resources to get this, and i will review your pr and guide you if you make errors
